src/main/java/com/google/flourbot/basic/Bot.java

```diff
@@ -54,128 +54,131 @@
 import java.security.GeneralSecurityException;
 import java.util.Collections;
 import java.util.ArrayList;
+import java.util.Arrays;
```

Comment:
comment for line 16, can't add it there in GitHub :/

The package should be .flourbot.api